.travis.yml

```diff
@@ -56,7 +55,11 @@ matrix:
     - os: linux
       arch: amd64
       env: TR_ARCH=podman-test
-      dist: bionic
+      dist: xenial # latest bionic kernel breaks this test case
+    - os: linux
+      arch: amd64
+      env: TR_ARCH=docker-test
+      dist: xenial # latest bionic kernel breaks this test case
```

Comment:
it seems like `openj9-test` fails because of the issue described in https://github.com/checkpoint-restore/criu/issues/860. The following would fix this test as well:
```diff
diff --git a/.travis.yml b/.travis.yml
index 71ede34ea..c35e4650e 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,6 @@ env:
   - TR_ARCH=local       CLANG=1 COMPAT_TEST=y
   - TR_ARCH=x86_64
   - TR_ARCH=x86_64      CLANG=1
-  - TR_ARCH=openj9-test
 matrix:
   include:
     - os: linux
@@ -60,6 +59,10 @@ matrix:
       arch: amd64
       env: TR_ARCH=docker-test
       dist: xenial # latest bionic kernel breaks this test case
+    - os: linux
+      arch: amd64
+      env: TR_ARCH=openj9-test
+      dist: xenial # latest bionic kernel breaks this test case
     - os: linux
       arch: amd64
       env: TR_ARCH=alpine      CLANG=1
```